.gitignore

```
@@ -3,7 +3,7 @@ node_modules
 AwesomeProject.xcodeproj
 AwesomeProjectTests
 index.ios.js
-iOS
+xcuserdata
 
 .idea/
 .vscode/
```

Github is warning us of a missing new line at the end of the file :)
```suggestion
.vscode/

```